src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for Phone, Name, Email and Address class in the addressbook 
+ * so that each class produces a printable string representation of the object.
+ */
+public interface Printable {
+    public String getPrintableString();
+}
```

Comment:
You have not implemented the second part of the tutorial activity. 
You must implement a method String getPrintableString(Printable... printables){...}  is a suitable place to print a list of details.